### PR TITLE
Beta 14 Release

### PR DESCRIPTION
**Pull request by fubuloubu**
Please add suggestions to what we should mention in the release notes for b14.

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
+
+Date released: TBD
+
+The following VIPs were implemented for Beta 12:
+
+- 
+
+Some of the bug and stability fixes:
+
+- Python 3.8 support
```
Comment (fubuloubu) [inline docs/release-notes.rst]:
```suggestion
- Mucho Documentation and Example cleanup!
- Python 3.8 support (`#1678 <https://github.com/ethereum/vyper/pull/1678>`_)
```

File: docs/release-notes.rst [hunk 2]
```diff
@@ -3,6 +3,19 @@
...
+
```
Comment (fubuloubu) [inline docs/release-notes.rst]:
```suggestion
- Disallow scientific notation in literals, which previously parsed incorrectly (`#1681 <https://github.com/ethereum/vyper/pull/1681>`_)
- Add implicit rewrite rule for `bytes[32]` -> `bytes32` (`#1718 <https://github.com/ethereum/vyper/pull/1718>`_)
- Support `bytes32` in `raw_log` (`#1719 <https://github.com/ethereum/vyper/pull/1719>`_)
- Fixed EOF parsing bug (`#1720 <https://github.com/ethereum/vyper/pull/1720>`_)
- Cleaned up arithmetic expressions (`#1661 <https://github.com/ethereum/vyper/pull/1661>`_)
- Fixed off-by-one in check for homogeneous list element types (`#1673 <https://github.com/ethereum/vyper/pull/1673>`_)
- Fixed stack valency issues in if and for statements (`#1665 <https://github.com/ethereum/vyper/pull/1665>`_)
- Prevent overflow when using `sqrt` on certain datatypes (`#1679 <https://github.com/ethereum/vyper/pull/1679>`_)
- Prevent shadowing of internval variables (`#1601 <https://github.com/ethereum/vyper/pull/1601>`_)
- Reject unary substraction on unsigned types  (`#1638 <https://github.com/ethereum/vyper/pull/1638>`_)
- Disallow `orelse` syntax in `for` loops (`#1633 <https://github.com/ethereum/vyper/pull/1633>`_)
- Increased clarity and efficiency of zero-padding (`#1605 <https://github.com/ethereum/vyper/pull/1605>`_)

```

File: docs/release-notes.rst [hunk 3]
```diff
@@ -3,6 +3,19 @@
...
 
+v0.1.0-beta.14
+**************
+
+Date released: TBD
+
+The following VIPs were implemented for Beta 12:
+
+- 
+
```
Comment (fubuloubu) [inline docs/release-notes.rst]: No VIPs were implemented

File: docs/release-notes.rst [hunk 4]
```diff
@@ -3,6 +3,28 @@
...
+- Python 3.8 support (`#1678 <https://github.com/ethereum/vyper/pull/1678>`_)
+- Disallow scientific notation in literals, which previously parsed incorrectly (`#1681 <https://github.com/ethereum/vyper/pull/1681>`_)
+- Add implicit rewrite rule for `bytes[32]` -> `bytes32` (`#1718 <https://github.com/ethereum/vyper/pull/1718>`_)
+- Support `bytes32` in `raw_log` (`#1719 <https://github.com/ethereum/vyper/pull/1719>`_)
+- Fixed EOF parsing bug (`#1720 <https://github.com/ethereum/vyper/pull/1720>`_)
+- Cleaned up arithmetic expressions (`#1661 <https://github.com/ethereum/vyper/pull/1661>`_)
+- Fixed off-by-one in check for homogeneous list element types (`#1673 <https://github.com/ethereum/vyper/pull/1673>`_)
+- Fixed stack valency issues in if and for statements (`#1665 <https://github.com/ethereum/vyper/pull/1665>`_)
+- Prevent overflow when using `sqrt` on certain datatypes (`#1679 <https://github.com/ethereum/vyper/pull/1679>`_)
+- Prevent shadowing of internval variables (`#1601 <https://github.com/ethereum/vyper/pull/1601>`_)
```
Comment (iamdefinitelyahuman) [inline docs/release-notes.rst]:
```suggestion
- Prevent shadowing of internal variables (`#1601 <https://github.com/ethereum/vyper/pull/1601>`_)
```

File: docs/release-notes.rst [hunk 5]
```diff
@@ -3,6 +3,28 @@
...
+**************
+
+Date released: TBD
+
+Some of the bug and stability fixes:
+
+- Mucho Documentation and Example cleanup!
+- Python 3.8 support (`#1678 <https://github.com/ethereum/vyper/pull/1678>`_)
+- Disallow scientific notation in literals, which previously parsed incorrectly (`#1681 <https://github.com/ethereum/vyper/pull/1681>`_)
+- Add implicit rewrite rule for `bytes[32]` -> `bytes32` (`#1718 <https://github.com/ethereum/vyper/pull/1718>`_)
```
Comment (iamdefinitelyahuman) [inline docs/release-notes.rst]:
```suggestion
- Add implicit rewrite rule for ``bytes[32]`` -> ``bytes32`` (`#1718 <https://github.com/ethereum/vyper/pull/1718>`_)
```

File: docs/release-notes.rst [hunk 6]
```diff
@@ -3,6 +3,28 @@
...
+- Support `bytes32` in `raw_log` (`#1719 <https://github.com/ethereum/vyper/pull/1719>`_)
```
Comment (iamdefinitelyahuman) [inline docs/release-notes.rst]:
```suggestion
- Support ``bytes32`` in ``raw_log`` (`#1719 <https://github.com/ethereum/vyper/pull/1719>`_)
```

File: docs/release-notes.rst [hunk 7]
```diff
@@ -3,6 +3,28 @@
...
+- Fixed EOF parsing bug (`#1720 <https://github.com/ethereum/vyper/pull/1720>`_)
+- Cleaned up arithmetic expressions (`#1661 <https://github.com/ethereum/vyper/pull/1661>`_)
+- Fixed off-by-one in check for homogeneous list element types (`#1673 <https://github.com/ethereum/vyper/pull/1673>`_)
+- Fixed stack valency issues in if and for statements (`#1665 <https://github.com/ethereum/vyper/pull/1665>`_)
+- Prevent overflow when using `sqrt` on certain datatypes (`#1679 <https://github.com/ethereum/vyper/pull/1679>`_)
```
Comment (iamdefinitelyahuman) [inline docs/release-notes.rst]:
```suggestion
- Prevent overflow when using ``sqrt`` on certain datatypes (`#1679 <https://github.com/ethereum/vyper/pull/1679>`_)
```

File: docs/release-notes.rst [hunk 8]
```diff
@@ -3,6 +3,28 @@
...
+- Prevent shadowing of internval variables (`#1601 <https://github.com/ethereum/vyper/pull/1601>`_)
+- Reject unary substraction on unsigned types  (`#1638 <https://github.com/ethereum/vyper/pull/1638>`_)
+- Disallow `orelse` syntax in `for` loops (`#1633 <https://github.com/ethereum/vyper/pull/1633>`_)
```
Comment (iamdefinitelyahuman) [inline docs/release-notes.rst]:
```suggestion
- Disallow ``orelse`` syntax in ``for`` loops (`#1633 <https://github.com/ethereum/vyper/pull/1633>`_)
```

File: docs/release-notes.rst [hunk 9]
```diff
@@ -3,6 +3,28 @@
 Release Notes
 #############
 
+v0.1.0-beta.14
+**************
+
+Date released: TBD
```
Comment (fubuloubu) [inline docs/release-notes.rst]: Release tomorrow?
```suggestion
Date released: 13-11-2019
```

**Pull request merged**